Tests/ValetTests/CloudAccessibilityTests.swift

```diff
@@ -0,0 +1,37 @@
+//  Created by Dan Federman on 1/20/20.
+//  Copyright © 2020 Square, Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+import XCTest
+
+@testable import Valet
+
+
+final class CloudAccessibilityTests: XCTestCase {
+
+    func test_description_mirrorsAccessibilityCounterpartDescription() {
+        CloudAccessibility.allValues().forEach {
+            XCTAssertEqual($0.description, $0.accessibility.description)
+        }
+    }
+
+    func test_secAccessibilityAttribute_mirrorsAccessibilityCounterpartSecAccessibilityAttribute() {
+        CloudAccessibility.allValues().forEach {
```

Comment:
Should we make `CloudAccessibility` `CaseIterable`?

Reply:
I’ve avoided doing that in the past, since I wanted to limit the possibility that adding a new accessibility type would be a source-breaking change. (Yes, someone COULD switch over the entire accessibility enum, but that seems less likely than enumerating all cases?). 

Note that `allValues` is an `internal` property.

That said, I’m open to `CaseIterable`, since I think I’m convincing myself adding a new case here is already a source-breaking change because of switch statements. Thoughts?

Reply:
(We can tackle `CaseIterable` in one fell swoop on all of our enums if we decide to go that way. I still want to know your thoughts!)

Reply:
I agree that adding a new case will already be a source-breaking change even without the `CaseIterable`. This is something that the [frozen enums](https://github.com/apple/swift-evolution/blob/master/proposals/0192-non-exhaustive-enums.md) change tried to address (but it's not implemented _yey_ for Swift enums), so IMO adding the convenience `CaseIterable` doesn't modify much but gives us the benefit of not having to maintain the allValues manually which is error prone.

Reply:
Makes sense. I'll put out a PR once the other outstanding PRs are merged, since doing so now would cause conflicts.